### PR TITLE
add arabic language

### PR DESCRIPTION
**Pull request by Hamad3bdulla**

### CODE REVIEW

File: src/lib/i18n/locales/languages.json
Comment (justinh-rahb) [inline src/lib/i18n/locales/languages.json:1]: Should be below the default option.

**Comment by justinh-rahb** [comment]
Thanks for this additional language PR! In keeping with convention, we organize the list alphabetically by the code, with the exception of the default `en-US` being at the top.

File: src/lib/i18n/locales/languages.json
Comment (bo3bdo) [reply src/lib/i18n/locales/languages.json:1]: i add it below en-US ? 
I see English is the default

File: src/lib/i18n/locales/languages.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 [
+	{
+		"code": "ar-BH",
+		"title": "العربية"
```
Comment (justinh-rahb) [inline src/lib/i18n/locales/languages.json:4]:
```json
	{
		"code": "ar-BH",
		"title": "العربية (Arabic)"
```

**Comment by bo3bdo** [comment]
oh ok i update and add new one to dev

**Pull request closed without merging**